rep-2000.rst

```diff
@@ -1185,6 +1185,121 @@ Build System Support:
 - cmake
 - setuptools
 
+Kilted Kaiju (May 2025 - November 2026)
+---------------------------------------
+
+Targeted platforms:
+
++--------------+------------------+---------------+------------------+-------------+-----------------+----------------+
+| Architecture | Ubuntu Noble     | Windows 10    | RHEL 9           |  macOS      | Debian Bookworm | OpenEmbedded / |
+|              | (24.04)          | (VS2019)      |                  |             | (12)            | Yocto Project  |
++==============+==================+===============+==================+=============+=================+================+
+|    amd64     | Tier 1 [d][a][s] | Tier 1 [a][s] | Tier 2 [d][a][s] |  Tier 3 [s] |   Tier 3 [s]    |   Tier 3 [s]   |
++--------------+------------------+---------------+------------------+-------------+-----------------+----------------+
+|    arm64     | Tier 1 [d][a][s] |               |                  |             |   Tier 3 [s]    |   Tier 3 [s]   |
++--------------+------------------+---------------+------------------+-------------+-----------------+----------------+
+|    arm32     |    Tier 3 [s]    |               |                  |             |   Tier 3 [s]    |   Tier 3 [s]   |
++--------------+------------------+---------------+------------------+-------------+-----------------+----------------+
+
+The following indicators show what delivery mechanisms are available for each platform.
+
+" [d] " Distribution-specific (Debian, RPM, etc.) packages will be provided for this platform for packages submitted to the rosdistro.
+
+" [a] " Binary releases are provided as a single archive per platform containing all packages in the Jazzy ROS 2 repos file [14]_.
+
+" [s] " Compilation from source.
+
+Middleware Implementation Support:
+
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
+| Middleware Library       | Middleware Provider    | Support Level | Platforms                  | Architectures                  |
++==========================+========================+===============+============================+================================+
+| rmw_fastrtps_cpp         | eProsima Fast-DDS      | Tier 1        | All Platforms              | All Architectures              |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
```

Comment:
We typically list the default middleware first. Should we enforce any ordering on the other Tier 1 providers?

```suggestion
+--------------------------+------------------------+---------------+----------------------------+--------------------------------+
| rmw_zenoh_cpp            | Eclipse Zenoh          | Tier 1        | All Platforms              | All Architectures              |
+--------------------------+------------------------+---------------+----------------------------+--------------------------------+
```

Reply:
I would say primary sort is by default first (so `rmw_fastrtps_cpp`).  Secondary sort by Tier level.  Tertiary sort by name.

With that in mind, I think this list should be:

1. `rmw_fastrtps_cpp`
2. `rmw_connextdds`
3. `rmw_cyclonedds_cpp`
4. `rmw_zenoh_cpp`
5. `rmw_fastrtps_dynamic_cpp`
6. `rmw_gurumdds_cpp`

Reply:
Done in 6b1c523. I also added Zenoh to the "RMW DDS Middleware" section later on. Should we rename that section to simply "RMW Middleware" or maybe "RMW Middleware Providers" or something?

Also, I simply alphabetized that section. I don't see any value in showing the default provider first in that context.

Reply:
> Done in [6b1c523](https://github.com/ros-infrastructure/rep/commit/6b1c5234c6707ee9318014b9481a3904e5bee73d). I also added Zenoh to the "RMW DDS Middleware" section later on. Should we rename that section to simply "RMW Middleware" or maybe "RMW Middleware Providers" or something?

Yeah, good call.  I'll add a comment there to change that.